src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -860,5 +894,75 @@ public void CZ_CHECK_PING(ChannelConnection conn, Packet packet)
 		{
 			// No parameters, no response.
 		}
+
+		/// <summary>
+		/// This packet is used to cast skills in the ground
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		/// <example>
+		/// 
+		/// </example>
+		[PacketHandler(Op.CZ_SKILL_GROUND)]
+		public void CZ_SKILL_GROUND(ChannelConnection conn, Packet packet)
+		{
+			var unk1 = packet.GetByte();
+			var skillId = packet.GetInt();
+			var unk2 = packet.GetInt();
+			var x1 = packet.GetFloat();
+			var y1 = packet.GetFloat();
+			var z1 = packet.GetFloat();
+			var x2 = packet.GetFloat();
+			var y2 = packet.GetFloat();
+			var z2 = packet.GetFloat();
+			var cos = packet.GetFloat();
+			var sin = packet.GetFloat();
+			var unk3 = packet.GetInt(); // This seems to be "target actorId"
+			var unk6 = packet.GetByte();
+			var unk7 = packet.GetByte();
+
+			Log.Warning("CZ_SKILL_GROUND");
```

Comment:
Same as above. I realize I haven't specified this anywhere for Melia yet, master is supposed to be the clean and stable branch. There should be no unnecessary testing code, or random warnings that don't mean anything to the user. To work on unfinished features in peace, until they're ready, we should utilize branches.